internal/localizer/localizer.go

```diff
@@ -0,0 +1,109 @@
+package localizer
+
+import (
+	// Import the internal/translations so that it's init() function
+	// is run. It's really important that we do this here so that the
+	// default message catalog is updated to use our translations
+	// *before* we initialize the message.Printer instances below.
+
+	"fmt"
+	"os"
+
+	_ "github.com/microsoft/go-sqlcmd/internal/translations"
+	"golang.org/x/text/language"
+	"golang.org/x/text/message"
+)
+
+var Translator Localizer
+
+// Define a Localizer type which stores the relevant locale ID and a
+// (deliberately unexported) message.Printer instance for the locale.
+type Localizer struct {
+	ID      string
+	printer *message.Printer
+}
+
+// Initialize a slice which holds the initialized Localizer types for
+// each of our supported locales.
+var locales = []Localizer{
+	{
+		ID:      "de-de",
+		printer: message.NewPrinter(language.MustParse("de-DE")),
+	},
+	{
+		ID:      "fr-fr",
+		printer: message.NewPrinter(language.MustParse("fr-FR")),
+	},
+	{
+		ID:      "en-us",
+		printer: message.NewPrinter(language.MustParse("en-US")),
+	},
+	{
+		ID:      "zh-cn",
```

Comment:
>cn", [](http://example.com/codeflow?start=15&length=4)

the modern lang ids are zh-hans and zh-hant

Reply:
Added modern lang ids as suggested.